enzy_htp/structure/chain.py

```diff
@@ -44,13 +42,15 @@ def __init__(self, name: str, residues: List[Residue], parent=None):
     @property
     def name(self) -> str:
         return self._name
+
```

Comment:
I would like to keep those method pairs together without an empty line in between. But it is fine if this is done by yapf automatically.

Reply:
yeah this is from YAPF